Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -158,6 +124,7 @@ - (void)openSimulatorHeadlessWithCompletion:(void (^)(NSError *))completion {
 - (NSError *)waitForDeviceReady {
     int attempts = 1200;
     while (attempts > 0 && ![self.device.stateString isEqualToString:@"Booted"]) {
+        [BPUtils printInfo:INFO withString:@"Simulator %@ currently has the state: %@", self.device.UDID.UUIDString, self.device.stateString];
```

Comment:
This seems like it's unnecessary logging. If you want to leave it I'd at least make it `DEBUGINFO`.